app/upgrades.go

```diff
@@ -58,109 +39,43 @@ var CircuitSuperAdmins = map[string][]string{
-			keyTable.RegisterParamSet(&ibcconnectiontypes.Params{})
-		case ibctransfertypes.ModuleName:
-			keyTable = ibctransfertypes.ParamKeyTable()
-		case icacontrollertypes.SubModuleName:
-			keyTable = icacontrollertypes.ParamKeyTable()
-		case icahosttypes.SubModuleName:
-			keyTable = icahosttypes.ParamKeyTable()
-		default:
-			continue
-		}
-		if !subspace.HasKeyTable() {
-			subspace.WithKeyTable(keyTable)
-		}
-	}
-	baseAppLegacySS := app.ParamsKeeper.Subspace(baseapp.Paramspace).WithKeyTable(paramstypes.ConsensusParamsKeyTable())
+	planName := "v7.0.0"
 
 	app.UpgradeKeeper.SetUpgradeHandler(planName, func(ctx context.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
 		sdkCtx := sdk.UnwrapSDKContext(ctx)
 
-		// OPTIONAL: prune expired tendermint consensus states to save storage space
-		if _, err := ibctmmigrations.PruneExpiredConsensusStates(sdkCtx, cdc, app.IBCKeeper.ClientKeeper); err != nil {
-			return nil, err
-		}
-
-		// explicitly update the IBC 02-client params, adding the localhost client type
-		var allowedClients []string
-		ibcParamSpace := app.GetSubspace(ibcexported.ModuleName)
-		ibcParamSpace.Get(sdkCtx, ibcclienttypes.KeyAllowedClients, &allowedClients)
-		allowedClients = append(allowedClients, ibcexported.Localhost)
-		ibcParamSpace.SetParamSet(sdkCtx, &ibcclienttypes.Params{
-			AllowedClients: allowedClients,
-		})
-		if err := ibcmigrationsv6.MigrateICS27ChannelCapability(
-			sdkCtx,
-			cdc,
-			app.keys["capability"],
-			app.CapabilityKeeper,
-			icacontrollertypes.SubModuleName,
-		); err != nil {
-			return nil, err
-		}
-
-		// Migrate Tendermint consensus parameters from x/params module to a dedicated x/consensus module.
-		if err := baseapp.MigrateParams(sdkCtx, baseAppLegacySS, &app.ConsensusParamsKeeper.ParamsStore); err != nil {
-			return nil, err
-		}
 		sdkCtx.Logger().Info("start to run module migrations...")
 
 		m, err := app.ModuleManager.RunMigrations(ctx, app.configurator, fromVM)
 		if err != nil {
 			return map[string]uint64{}, err
 		}
-		{
-			params := app.ICAHostKeeper.GetParams(sdkCtx)
-			msg := "/ibc.applications.interchain_accounts.host.v1.MsgModuleQuerySafe"
-			if !slices.ContainsFunc(params.AllowMessages, func(allowMsg string) bool {
-				return allowMsg == "*" || allowMsg == msg
-			}) {
-				params.AllowMessages = append(params.AllowMessages, msg)
-				app.ICAHostKeeper.SetParams(sdkCtx, params)
-			}
-			if err := UpdateExpeditedParams(ctx, app.GovKeeper); err != nil {
-				return map[string]uint64{}, err
-			}
+
+		var _ok bool
+		maxSupplyParams := maxsupplytypes.DefaultParams()
+		// update max supply to 100B * 10^18 basecro
+		maxSupplyParams.MaxSupply, _ok = sdkmath.NewIntFromString("100000000000000000000000000000")
+		if !_ok {
+			return map[string]uint64{}, fmt.Errorf("invalid max supply")
+		}
+		maxSupplyParams.BurnedAddresses = []string{
+			"cro1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqtcgxmv",
```

Comment:
may need to check if we are on testnet or mainnet
fyi burned addresses are : 
Mainnet: cro1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqtcgxmv
Testnet: tcro1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq9dpzma